src/reanimated2/js-reanimated/JSReanimated.ts

```diff
@@ -13,9 +13,10 @@ export default class JSReanimated {
   timeProvider = null;
 
   constructor() {
-    this.timeProvider = process.env.JEST_WORKER_ID
-      ? { now: () => Date.now() }
-      : window.performance;
+    this.timeProvider = {
+      now: () =>
+        process.env.JEST_WORKER_ID ? Date.now() : window.performance.now(),
+    };
```

Comment:
`timeProvider.now()` is called in every frame on the web implementation, so could you change this to a little bit faster version without `if` condition? Something like this:
```js
this.timeProvider = {};
if(process.env.JEST_WORKER_ID) {
  this.timeProvider.now = () => Date.now();
}
else {
  this.timeProvider.now = () => window.performance.now();
}
```

Reply:
Yeah that makes more sense, I went with a ternary to stay consistent. Thanks for taking this on and making the fix!